app/policies/course_lesson_part_policy.rb

```diff
@@ -25,4 +25,12 @@ def show_split?
   def split?
     update?
   end
+
+  def show_delete?
+    update?
+  end
+
+  def destroy?
+    update?
+  end
```

Comment:
`show_delete` should probably be the same as `destroy`, rather than `update`. 

And, more importantly, we probably don't want to let the user delete the last lesson part - that would lead to some bad times. In here we should check that the `@record.course_lesson.course_lesson_parts.length > 1` or something like that, and if not, then return false.